buildSrc/src/main/java/dependencies/Versions.kt

```diff
@@ -3,4 +3,11 @@ package dependencies
 private object Versions {
     val androidCompileSdkVersion = 28
     val androidMinSdkVersion = 21
+
+    private val versionMajor = 1
+    private val versionMinor = 0
+    private val versionPatch = 0
+    val androidVersionCode = versionMajor * 10000 + versionMinor * 100 + versionPatch
+
+    val androidVersionName = "1.0.0"
```

Comment:
ah, sorry I missed it before! what about using the versionMajor/minor/patch here as well?
``` kotlin
val androidVersionName = "$versionMajor.$versionMinor.$versionPatch"
```

Reply:
Awesome💕